tembo-olap/README.md

```diff
@@ -0,0 +1,66 @@
+# Instantiation
+
+Select OLAP Stack from https://cloud.tembo.io
+
+## Choose System Configuration:
+
+**CPU**: 16vCPU
```

Comment:
I was unable to find any instance with 16 vCPU in tembo cloud.

Reply:
Hi @chhetripradeep ,
Thanks for looking into this pull request. Try now, earlier it wasn't enabled in the public domain. Added some more information regarding Instance type in the README.md file. Apologies for the inconvenience.